microprofile-openapi/README.adoc

```diff
@@ -20,28 +20,15 @@ To complete this guide, you will need:
 * JDK 1.8+ installed with `JAVA_HOME` configured appropriately
 * Apache Maven 3.5.3+
 
+include::../shared-doc/use-of-jboss-home-name.adoc[leveloffset=+1]
+ 
 == Steps
 
-. Start a {productName} instance.
+. Start a {productName} instance using the microprofile profile.
 +
 [source, shell]
 ----
-$ mvn wildfly:run &
-----
-
-. Add support for MicroProfile OpenAPI by adding the requisite extension and subsystem.
-+
-[source, shell]
-----
-$ mvn wildfly:execute-commands -Dwildfly.commands=/extension=org.wildfly.extension.microprofile.openapi-smallrye:add
-$ mvn wildfly:execute-commands -Dwildfly.commands=/subsystem=microprofile-openapi-smallrye:add
-----
-
-. Since the MicroProfile OpenAPI subsystem modifies {productName}'s deployment chain, the server needs to be reloaded.
-+
-[source, shell]
-----
-$ mvn wildfly:shutdown -Dwildfly.reload=true
+$ {jbossHomeName}/bin/standalone.sh --server-config=standalone-microprofile.xml &
 ----
 
 . Build and deploy the sample application to {productName}.
```

Comment:
instead this should be:

// Build and Deploy the Quickstart
include::../shared-doc/build-and-deploy-the-quickstart.adoc[leveloffset=+1]

and after the content wrt using/accessing the app:

// Undeploy the Quickstart
include::../shared-doc/undeploy-the-quickstart.adoc[leveloffset=+1]

Reply:
Done.